src/js/MousePositionController.js

```diff
@@ -14,7 +14,7 @@ goog.require('ga_measure_service');
 
     var coordinatesFormat = function(coordinates) {
       return $translate.instant('coordinates_label') + ': ' +
-          gaMeasure.formatCoordinates(coordinates);
+          gaMeasure.formatCoordinates(coordinates, 1).replace(/\.0/g,'');
```

Comment:
This replacement must go inside the [gaMeasure service](https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/measure/MeasureService.js#L23